content/node-runners/setup/docker.md

```diff
@@ -13,14 +13,10 @@ This makes it easy to run applications without worrying about the operating syst
 
 
 ```bash
-docker run --cap-add NET_ADMIN -d -p 4449:4449 --name myst -v $YOUR_MYSTERIUM_DIR:/var/lib/mysterium-node mysteriumnetwork/myst:latest service --agreed-terms-and-conditions wireguard
```

Comment:
I suggest to change `$YOUR_MYSTERIUM_DIR` to `myst-data` to make it work out of the box not confusing people. Docker will create the volume if it's missing or will be using the existing one if possible.

Reply:
Another note: we probably don't need to specify services. Let's keep it default to make configuration easy. 
If we need to disable openvpn by default, let's make it on the node level.